core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -146,8 +143,13 @@ public void init() throws StoreInitializationException {
     props.put(ProducerConfig.RETRIES_CONFIG, 0); // Producer should not retry
     producer = new KafkaProducer<byte[],byte[]>(props);
 
-    // start the background thread that subscribes to the Kafka topic and applies updates
-    kafkaTopicReader.start();
+    // start the background thread that subscribes to the Kafka topic and applies updates.
+    // the thread must be created after the schema topic has been created.
+    this.kafkaTopicReader =
+            new KafkaStoreReaderThread<>(this.bootstrapBrokers, topic, groupId,
+                    Integer.MIN_VALUE, this.storeUpdateHandler,
```

Comment:
It may have been an oversight that we ended up even exposing `KAFKASTORE_COMMIT_INTERVAL_MS_CONFIG` since commits are not used at all.

Unless we implement some sort of snapshot mechanism or use a backing store with that capability, it's probably better to get rid of this config altogether since it does nothing. Which means we continue to do what we already do - rebuild the in-memory caches by consuming from offset 0 if we restart the process. 

I think the reasoning here is that the number of schemas in the lifetime of an organization is small enough that building from t=0 on startup is ok.